sound/soc/sof/intel/hda-loader.c

```diff
@@ -78,26 +78,14 @@ static int cl_stream_prepare(struct snd_sof_dev *sdev, unsigned int format,
  * reset/stall and then turn it off
  */
 static int cl_dsp_init(struct snd_sof_dev *sdev, const void *fwdata,
-		       u32 fwsize)
+		       u32 fwsize, int tag)
 {
-	int tag, ret, i;
+	int ret, i;
 	u32 hipcie;
 	struct sof_intel_hda_dev *hda =
 		(struct sof_intel_hda_dev *)sdev->pdata->hw_pdata;
```

Comment:
maybe something for a separate clean up patch: the above cast from a "void *" pointer is superfluous